src/group_theory/subgroup.lean

```diff
@@ -1171,6 +1171,10 @@ def ker (f : G →* N) := (⊥ : subgroup N).comap f
 @[to_additive]
 lemma mem_ker (f : G →* N) {x : G} : x ∈ f.ker ↔ f x = 1 := iff.rfl
 
+instance decidable_ker [decidable_eq N] (f : G →* N) :
```

Comment:
I think this statement should resemble the one of the existing `decidable_mem_range`. Right now it doesn't use `\mem`, and does use `carrier`.

Reply:
Unfortunately, the format of `subgroup.fintype` and the format of `decidable_mem_range` are incompatible, and proving `subtype.fintype` throws in a `coe` that makes the whole thing unusable... perhaps I can refactor `subgroup.fintype`.

Reply:
Ok, I found something that probably works, let's see if it breaks any other files...